app/assets/stylesheets/application.scss

```diff
@@ -15,3 +15,25 @@
  *= require foundation_and_overrides
 
  */
+@import url(https://fonts.googleapis.com/css?family=Alfa+Slab+One);
+
+.home-page {
+  background: url('/assets/panama_lineup.jpg');
+  background-size: cover;
+  height: 100%;
+  width: 100%;
+}
+
+.welcome {
+  background-color: rgba(0, 0, 0, 0.3);
+  border: 1px solid #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.